src/app/stores/file.ts

```diff
@@ -109,4 +112,30 @@ export class FileStore implements AppStore {
   }
 
   stop() { }
-}
\ No newline at end of file
+}
+
+function getLocalSecrets(cwd = process.cwd()) {
+  const localSecretsPath = path.join(cwd, secretsFile)
+  let secrets = {};
+
+  if (fs.existsSync(localSecretsPath))
+    secrets = YAML.load(fs.readFileSync(localSecretsPath).toString())
+
+  return secrets
+}
+
+function getLocalConfig(
+  cwd = process.cwd(),
+  env = getEnv()
+) {
+  const localConfigPath = path.join(cwd, configFile)
+  let config: any = {};
+  if (fs.existsSync(localConfigPath))
+    config = YAML.load(fs.readFileSync(localConfigPath).toString())
+
+  return config[env] || config
+}
```

Comment:
Nice, these 2 make more sense in this file.

Reply:
sadly, deploy command needs them. onto more refactoring